src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocksNoViolations.java

```diff
@@ -5,99 +5,92 @@
 
 import java.io.IOException;
 
-public class InputEmptyBlocksAndCatchBlocksNoViolations
-{
```

Comment:
I assume the formatter is changing this. Is it possible to force the `{` to stay on this line like if we added a comment on the end of the previous line or made the class name excessively long?

I don't want any changes from this one. It is more a question right now.

Reply:
>Is it possible to force the `{` to stay on this line like if we added a comment on the end of the previous line

Yes, in this case the `{` stays on the new line even after formatting though the `{` indentation increases by 1:

```java
public class InputEmptyBlocksAndCatchBlocksNoViolations // testing
 {
```

> or made the class name excessively long?

In this case, the formatter adds a line break after `public` and keeps the `{` on the same line:

```java
public
class InputEmptyBlocksAndCatchBlocksNoViolationssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss {
```

Reply:
Formatter is not configurable. 
in this section Inputs we do not care about `{` location so we can follow formatter to be "google style friendly by all means"